devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -203,17 +217,34 @@ private getButtonMap() {[
 				"02" : 3,
 				"03" : 1,
 				"04" : 2
-		]
+		] ,
+        "ADUROLIGHT_CSC" : [
+                "01" : 1,
+                "02" : 2,
+                "03" : 3,
+                "04" : 4
+        ],
+        "Adurolight_NCC" : [
+                "01" : 1,
+                "02" : 2,
+                "03" : 3,
+                "04" : 4
+        ]
 ]}
 
 private getSupportedButtonValues() {
 	def values = ["pushed", "held"]
+    if ((device.getDataValue("model") == "Adurolight_NCC") || (device.getDataValue("model") == "ADUROLIGHT_CSC")) {
+        values = ["pushed"]
+    }
 	return values
 }
 
 private getModelNumberOfButtons() {[
 		"3450-L" : 4,
-		"3450-L2" : 4
+		"3450-L2" : 4, 
+        "ADUROLIGHT_CSC" : 4,
+        "Adurolight_NCC" : 4
 ]}
 
 private getModelBindings(model) {
```

Comment:
I'm worried that since your devices report using the level control cluster and you're not setting up any bindings here that they may not actually report when they are pressed.

Reply:
same for 0xFCCC

Reply:
@adurosmart did you take a look at this comment? I think you at least need the level control binding and likely the 0xFCCC binding unless you handle that cluster automatically without a binding.

Reply:
@tpmanley  Thank you review, i add LEVEL_CONTROL_CLUSTER  bindings.